gammapy/astro/darkmatter/spectra.py

```diff
@@ -114,19 +143,15 @@ def channel(self, channel):
         else:
             self._channel = channel
 
-    @property
-    def table_model(self):
```

Comment:
Please keep this method for backwards compatibility. We have to deprecate it for 1-2 minor version before we can remove it. See https://docs.gammapy.org/1.1/development/dev_howto.html?highlight=deprecation#how-to-handle-api-breaking-changes